packages/node_modules/pouchdb-adapter-idb/src/index.js

```diff
@@ -700,31 +696,29 @@ function init(api, opts, callback) {
 
     // Do a few setup operations (in parallel as much as possible):
     // 1. Fetch meta doc
-    // 2. Check blob support
-    // 3. Calculate docCount
-    // 4. Generate an instanceId if necessary
-    // 5. Store docCount and instanceId on meta doc
+    // 2. Calculate docCount
+    // 3. Generate an instanceId if necessary
+    // 4. Store docCount and instanceId on meta doc
 
     var txn = idb.transaction([
       META_STORE,
-      DETECT_BLOB_SUPPORT_STORE,
       DOC_STORE
     ], 'readwrite');
 
     var storedMetaDoc = false;
     var metaDoc;
     var docCount;
-    var blobSupport;
     var instanceId;
 
     function completeSetup() {
-      if (typeof blobSupport === 'undefined' || !storedMetaDoc) {
+      if (!storedMetaDoc) {
+        guardedConsole('error', 'Meta document may not have been stored');
         return;
       }
       api._meta = {
         name: dbName,
         instanceId: instanceId,
-        blobSupport: blobSupport
+        blobSupport: true // Versions of Chrome < 43 supported IndexedDB, but not Blob
```

Comment:
We dont need this if its hard coded, just take out the falsy paths for anywhere that compares against this